microcloud/test/suites/preseed.sh

```diff
@@ -94,6 +97,7 @@ EOF
   lxc exec micro01 -- sh -c "
   cat << EOF > /root/preseed.yaml
 lookup_subnet: ${lookup_addr}/24
+lookup_interface: enp5s0
```

Comment:
Also are these really related to cephfs, shouldnt they be in a separate commit?

Reply:
Ah looks like you meant to include this change later after c45a983429b063a6070203606251ae1f1186f1ab

Reply:
Ah yeah oops that one belongs to a different commit. I included these here because the `preseed` tests were failing. 

I suppose we don't technically _need_ to require the interface name as well, but it does capture the edge case of 2 different interfaces sharing the same subnet or IP.

Since `microcoud init` will just present you with an entry for every address and every interface, it means if we don't include the interface name in `preseed` that it will actually be less precise than the interactive CLI at this step.